logic/src/commonMain/kotlin/com/wire/kalium/logic/data/prekey/PreKeyMapper.kt

```diff
@@ -1,16 +1,39 @@
 package com.wire.kalium.logic.data.prekey
 
-import com.wire.kalium.cryptography.PreKey
```

Comment:
This import change broke the `toPreKeyDTO` signature, which was expecting a `cryptography` `PreKey`.

This function was being used by `ClientMapper`.

We need to change `RegisterClientParam` to use the new `PreKey` class.

Reply:
yes I just finished from that now